app/src/main/java/com/fstyle/androidtrainning/screen/tabfavorite/TabFavoriteContract.java

```diff
@@ -1,7 +1,11 @@
 package com.fstyle.androidtrainning.screen.tabfavorite;
 
+import com.fstyle.androidtrainning.data.local.entity.MovieEntity;
+import com.fstyle.androidtrainning.data.model.Movie;
```

Comment:
ERROR: Unused import: com.fstyle.androidtrainning.data.model.Movie. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports